Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -375,16 +375,10 @@ fi
 PIP_USER_ARGUMENT="--user"
 
 # create a Python venv on more recent releases:
-PYTHON_VENV_PACKAGE=""
-if [ ${RELEASE_CODENAME} == 'bookworm' ] ||
-   [ ${RELEASE_CODENAME} == 'lunar' ] ||
-   [ ${RELEASE_CODENAME} == 'mantic' ]; then
-    PYTHON_VENV_PACKAGE=python3.11-venv
-elif [ ${RELEASE_CODENAME} == 'noble' ]; then
-    PYTHON_VENV_PACKAGE=python3.12-venv
-fi
+PYTHON_VENV_PACKAGE="python$(python3 --version | awk '{ print $2 }' | cut -d '.' -f 1,2)-venv"
+VENV_LOOKUP=$(apt search $PYTHON_VENV_PACKAGE 2>/dev/null| grep venv)
 
-if [ -n "$PYTHON_VENV_PACKAGE" ]; then
+if [[ $VENV_LOOKUP ]]; then
```

Comment:
There was a bug, which skipped installation of VENV on quite popular Ubuntu 22.04 code name jammy, which I fixed.
Since it was related to installation experience, I included it in this PR.